tests/testthat/test-CreatePackageVignette.R

```diff
@@ -0,0 +1,112 @@
+context("CreatePackageVignette Tests")
```

Comment:
do these tests fully cover `CreatePackageVignette()`? When I was reading the code I thought "wow this is a lot of if blocks, gonna take a lot of tests"

Reply:
Sort of. I have 100% test coverage according to `covr`.

I tested all of the error checks that use `if` blocks. 

I did not individually test failure for all of the `assertthat` checks.

Reply:
oh yeah definitely don't need to check all the individual `assertthat` blocks. If you caught all the `if` blocks I'm good!